app/src/main/java/net/squanchy/tweets/util/TwitterDateFormatter.java

```diff
@@ -23,11 +22,11 @@ private TwitterDateFormatter() {
 
     public static String getTimestampFrom(Tweet tweet, Context context) {
 
-        if (tweet.createdAt == null) {
+        if (tweet.createdAt() == null) {
```

Comment:
I think this can never happen, AutoValue would blow up if this were null as it's not marked as `@Nullable`—which is great, _if_ it is actually not mandatory I'd rather have it `Optional` than `@Nullable`